cns/restserver/homeazmonitor.go

```diff
@@ -62,6 +62,8 @@ func (h *HomeAzMonitor) readCacheValue() cns.GetHomeAzResponse {
 
 // Start starts a new thread to refresh home az cache
 func (h *HomeAzMonitor) Start() {
+	logger.Printf("[HomeAzMonitor] start the goroutine for refreshing homeAz")
+	defer h.Stop()
```

Comment:
i meant defer stop in main - if you do it here you will start and _immediately_ stop

Reply:
ohh.Thanks